Embark.xccolortheme

```diff
@@ -0,0 +1,232 @@
+		<key>xcode.syntax.regex.other</key>
+		<string>0.796078 0.890196 0.905882 1</string>
+		<key>xcode.syntax.string</key>
+		<string>1 0.901961 0.701961 1</string>
+		<key>xcode.syntax.url</key>
+		<string>0.330191 0.511266 0.998589 1</string>
+	</dict>
+	<key>DVTSourceTextSyntaxFonts</key>
+	<dict>
+		<key>xcode.syntax.attribute</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.character</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.comment</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.comment.doc</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.comment.doc.keyword</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.declaration.other</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.declaration.type</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.identifier.class</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.identifier.class.system</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.identifier.constant</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.identifier.constant.system</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.identifier.function</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.identifier.function.system</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.identifier.macro</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.identifier.macro.system</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.identifier.type</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.identifier.type.system</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.identifier.variable</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.identifier.variable.system</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.keyword</key>
+		<string>SFMono-Semibold - 15.0</string>
+		<key>xcode.syntax.mark</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.markup.code</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.number</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.plain</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.preprocessor</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.regex</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.regex.capturename</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.regex.charname</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.regex.number</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.regex.other</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.string</key>
+		<string>SFMono-Medium - 15.0</string>
+		<key>xcode.syntax.url</key>
+		<string>SFMono-Medium - 15.0</string>
```

Comment:
This is a font right? Wondering if it would be better for the theme to not do anything with fonts 

Thoughts?

Reply:
Yeah this font is real, this is an Apple font that is the default font for Xcode so it's a safe font to use as everybody has it installed.

This is normal and is the font every custom theme provides.

Reply:
I know nothing about xcode so I will trust your judgment. I just want to make sure these themes never overstep and go too far. Things like transparency, blur, line height, fonts, and other editor settings should be left stock and let the user take it from there. The themes should only be a color palette.

If you wanna throw a comment on the Readme about the settings of the screenshot feel free.

Reply:
Yeah I thought that as well so I created a copy of my theme and put it to stock everything so whoever uses it can adjust it to their liking with their preferred font and spacing. So if you're happy this can be pulled.